webpack.config.js

```diff
@@ -32,6 +32,19 @@ module.exports = {
                     presets: ['es2015', 'react', 'stage-1']
                 }
             },
+            {
+                test: /\.css$/,
+                exclude: /node_modules/,
+                loader: 'style-loader!css-loader',
+            },
+            {
+                test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
+                loader: "url-loader?limit=10000&minetype=application/font-woff"
+            },
+            {
+                test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
+                loader: "file-loader"
+            },
```

Comment:
@legastero, could you please add a new commit to this PR with comments on these 3 (like I have a comment bellow)?